### PR TITLE
feat: Support for new annotation for allowing use of hidden elements (issue #136)

### PR DESCRIPTION
**Pull request by hebbmcclure**
An implementation to add the requested feature of a `HiddenElement` annotation that will allow the bypass of `waitUntilDisplayed()`.

### CODE REVIEW

File: framework/src/main/java/io/github/kgress/scaffold/BaseWebElement.java [hunk 1]
```diff
@@ -143,8 +147,7 @@
    * @param cssSelector the string value of the {@link By#cssSelector(String)}
    */
   public BaseWebElement(String cssSelector) {
-    this.setBy(By.cssSelector(cssSelector));
```
Comment (Proryanator) [inline framework/src/main/java/io/github/kgress/scaffold/BaseWebElement.java:146]: You can restore these constructors to how they were before :)

Comment (hebbmcclure) [reply framework/src/main/java/io/github/kgress/scaffold/BaseWebElement.java:146]: Is there something wrong with DRYing them up a little?

File: framework/src/test/java/io/github/kgress/scaffold/webelement/HiddenElementTests.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      applyFieldAnnotations();
+    }
+
+    public TestBaseWebElement getHiddenThing() {
+      return hiddenThing;
+    }
+
+    public TestBaseWebElement getNotHiddenThing() {
+      return notHiddenThing;
+    }
```
Comment (Proryanator) [inline framework/src/test/java/io/github/kgress/scaffold/webelement/HiddenElementTests.java:40]: Small detail on this TestComponent, can add @Getter at the class level and strip out these methods.

**Comment by hebbmcclure** [comment]
Closing, Going to put up another with a different approach.

**Pull request closed without merging**